ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -446,10 +497,20 @@ public void blockMined(final Block block) {
     blockBroadcaster.propagate(block, totalDifficulty);
   }
 
-  public List<Bytes> getForkIdAsBytesList() {
-    final ForkId chainHeadForkId = forkIdManager.getForkIdForChainHead();
-    return chainHeadForkId == null
-        ? Collections.emptyList()
-        : chainHeadForkId.getForkIdAsBytesList();
+  private StatusMessage.BlockRange createBlockRange() {
+    return new StatusMessage.BlockRange(
+        blockchain
+            .getEarliest()
+            .map(
+                hash ->
+                    blockchain
+                        .getBlockHeader(hash)
+                        .orElseThrow(
+                            () ->
+                                new IllegalStateException(
+                                    "Unable to get earliest block header from blockchain."))
+                        .getNumber())
+            .orElse(0L),
```

Comment:
Should we set the right value if we have done checkpoint sync here?

Reply:
Removed